src/SearchResults.js

```diff
@@ -0,0 +1,61 @@
+/* I had help with task: L2 T15 */
+
+import React, { useState } from "react";
+import FakeBookings from "./data/fakeBookings.json";
+import moment from "moment";
+import "./SearchResults.css";
+
+const SearchResults = () => {
+  const [selectedRow, setSelectedRow] = useState(-1);
+
+  function rowSelected(index) {
+    if (index !== selectedRow) {
+      setSelectedRow(index);
+    } else {
+      setSelectedRow(-1);
+    }
+  }
```

Comment:
This is clever - I like the idea of setting `selectedRow` to -1 to "unselect".

However, if you read the exercise carefully it says that each row can be selected _independently_. So that means that you can select more than one row at once. Could you try implementing that in your app? (This is a tricky problem! We'll talk about it on Sunday if you're stuck!)